pycontw/pycon.tw
# fix(scss): show logos of job-listings page correctly

## Types of changes


## Description
The original width is insufficient to show some logos with larger width. This commit will unset the width restriction.

A bonus commit is to update `.gitignore`.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
Go to job-listings page to check the logos are shown correctly.

## Expected behavior
The width is sufficient to show a logo, e.g.
![Selection_220](https://user-images.githubusercontent.com/3217432/91477080-73b05900-e8d0-11ea-92de-8dcab687e5b7.png)

## More Information
**Screenshots**
The buggy frontend examples

![Selection_221](https://user-images.githubusercontent.com/3217432/91477155-8f1b6400-e8d0-11ea-8086-436b6ac82fd0.png)


This one is currently in the production site.

![Selection_222](https://user-images.githubusercontent.com/3217432/91477210-a0647080-e8d0-11ea-8def-18e005305c5f.png)
